### DISCUSSION TITLE
Private Offline Repository

### DISCUSSION THREAD

**Discussion by unix0r** [microsoft/winget-cli-restsource#115] [General]
Hey guys, first I wanted to thank you for this great project. It might be a good tool for our industry pc production at work.
For this I have some questions regarding the private repositories.

As the Readme describes for local running repositories, a cosmo database in azure is needed. As I know azure is the microsoft cloud solution and needs internet access.

I want to know if it is possible to run an offline repository with a local running database (not the azure cosmo)?
For example: 
A private network with a Windows Server running a private winget repository that may have internet access, deploys applications to computers in the private network that do not have internet access.

**Comment by mherrmann** [comment]
Hi @unix0r, we have a private repository at https://winget.pro that does not require Cosmo. We use it for our hosted offering but it can also run on-premise. In case you're interested in licensing it, you can reach out by email to "winget" at omaha-consulting.com.
